client/components/mma/accountoverview/CanadaStrike.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/react';
+import { Link } from '@guardian/source/react-components';
+import { ProblemAlert } from '../shared/ProblemAlert';
+
+export const CanadaStrike = () => (
+	<ProblemAlert
+		message={
+			<div>
+				<p>
+					Due to industrial action by Canada Post, it is not possible
+					to deliver your copies of the Guardian Weekly. You are
+					welcome to pause your subscription during the period of
+					industrial action, details on how to do so can be found{' '}
+					<Link
+						href="https://manage.theguardian.com/help-centre/article/i-need-to-pause-my-delivery"
+						priority="primary"
+					>
+						here
+					</Link>
+					.
+				</p>
+				<p>
+					If you have reached your allowance limit please contact{' '}
+					<Link
+						href="mailto:customer.help@theguardian.com"
+						priority="primary"
+					>
+						customer.help@guardian.co.uk
```

Comment:
why the difference in these email address domains?  I assume it should match the provided copy
```suggestion
						href="mailto:customer.help@guardian.co.uk"
						priority="primary"
					>
						customer.help@guardian.co.uk
```

Reply:
yep thanks for spotting that John 👍